doc/Makefile.am

```diff
@@ -148,6 +148,75 @@ CLEAN_LOCAL_TARGETS += clean-gtkdoc-header-local
 clean-gtkdoc-header-local:
 	-rm -rf xml/ Doxyfile-gi Doxyfile-gi.stamp geany-gtkdoc.h geany-sciwrappers-gtkdoc.h
 
+if HAVE_INTROSPECTION
+
+INTROSPECTION_GIRS  = Geany-1.0-pre.gir GeanyScintilla-1.0-pre.gir
```

Comment:
IIUC, intermediate files that get fixed afterward.  I guess `tmp` might make more sense